src/main/cli/cli.c

```diff
@@ -4957,7 +4955,7 @@ static void cliTasks(const char *cmdName, char *cmdline)
         cfCheckFuncInfo_t checkFuncInfo;
         getCheckFuncInfo(&checkFuncInfo);
         cliPrintLinef("RX Check Function %19d %7d %25d", checkFuncInfo.maxExecutionTimeUs, checkFuncInfo.averageExecutionTimeUs, checkFuncInfo.totalExecutionTimeUs / 1000);
-        cliPrintLinef("Total (excluding SERIAL) %25d.%1d%% %4d.%1d%%", maxLoadSum/10, maxLoadSum%10, averageLoadSum/10, averageLoadSum%10);
+        cliPrintLinef("Total (excluding SERIAL) %33d.%1d%%", averageLoadSum/10, averageLoadSum%10);
```

Comment:
Is there a reason for removing the `maxLoadSum` value from `tasks`?

Reply:
Because it's meaningless. It was the sum of the peak task loads seen since the `tasks` command was last run. But as these peaks didn't happen at the same time it adds no value. The peak load of each task is of interest, but not this total.